hacks/update-chart.sh

```diff
@@ -7,25 +7,26 @@ set -o pipefail
 LAST_SYNC_COMMIT=$(git log -n1 --grep '^\[upstream-sync\]' --pretty='format:%h' helm/)
 TMPDIR=$(mktemp -d)
 
-UPSTREAM_ZIP_URL=${UPSTREAM_ZIP_URL:-"https://github.com/helm/charts/archive/master.zip"}
-UPSTREAM_CHART_DIR=${UPSTREAM_CHART_DIR:-"charts-master/stable"}
-UPSTREAM_CHART_NAME=${UPSTREAM_CHART_NAME:-"prometheus-operator"}
+UPSTREAM_ZIP_URL=${UPSTREAM_ZIP_URL:-"https://github.com/prometheus-community/helm-charts/archive/main.zip"}
+UPSTREAM_CHART_DIR=${UPSTREAM_CHART_DIR:-"helm-charts-main/charts"}
+UPSTREAM_CHART_NAME=${UPSTREAM_CHART_NAME:-"kube-prometheus-stack"}
+LOCAL_CHART_NAME=${LOCAL_CHART_NAME:-"prometheus-operator-app"}
 
 KEEP_TMPDIR=${KEEP_TMPDIR:-''}
 
 clean_on_exit() {
   [[ -z "${KEEP_TMPDIR}" ]] && rm -rf ${TMPDIR}
 }
 
-curl -sL -o ${TMPDIR}/master.zip ${UPSTREAM_ZIP_URL}
-unzip -d ${TMPDIR}/ ${TMPDIR}/master.zip ${UPSTREAM_CHART_DIR}/${UPSTREAM_CHART_NAME}/*
+curl -sL -o ${TMPDIR}/main.zip ${UPSTREAM_ZIP_URL}
+unzip -d ${TMPDIR}/ ${TMPDIR}/main.zip ${UPSTREAM_CHART_DIR}/${UPSTREAM_CHART_NAME}/*
 
 
 UPSTREAM_VERSION=$(cat ${TMPDIR}/${UPSTREAM_CHART_DIR}/${UPSTREAM_CHART_NAME}/Chart.yaml | awk '/^version:/ { print $2 }')
 
-cp -a ${TMPDIR}/${UPSTREAM_CHART_DIR}/${UPSTREAM_CHART_NAME}/* helm/${UPSTREAM_CHART_NAME}-app/
+cp -a ${TMPDIR}/${UPSTREAM_CHART_DIR}/${UPSTREAM_CHART_NAME}/* helm/${LOCAL_CHART_NAME}/
 
-git add helm/${UPSTREAM_CHART_NAME}-app/
+git add helm/${LOCAL_CHART_NAME}/
```

Comment:
So we are keeping the previous name? I am not saying this is a bad idea, but we will have to maintain that for now

Reply:
My reasoning was to avoid confusion. I do think `kube-prometheus-stack` is a more accurate description but if I have this right, it would be a breaking change.